Demos/Apps/ThemeApp/ThemeCustomizer.razor.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Blazorise;
+using Microsoft.AspNetCore.Components;
+
+namespace ThemeApp
+{
+    public partial class ThemeCustomizer : ComponentBase
+    {
+        private string SelectedTab { get; set; } = "General";
```

Comment:
Move properties to end of class and enclose everything with regions. Example of code style https://github.com/stsrki/Blazorise/blob/master/Source/Blazorise/Badge.razor.cs

Reply:
Done